boot/makebootable.go

```diff
@@ -384,6 +385,18 @@ func useTokens(model *asserts.Model) bool {
 		// one installed, and will contain the same features.
 		return cryptsetupSupportsTokenReplace()
 	} else {
+		m, _ := kcmdline.KeyValues("ubuntu-core.force-experimental-tokens")
+		value, hasValue := m["ubuntu-core.force-experimental-tokens"]
+		if hasValue {
+			if value == "0" {
+				return false
+			}
+			if value == "1" {
+				return true
+			}
+			logger.Noticef("unexpected value for snapd.force-experimental-tokens")
+		}
+
```

Comment:
it's possible that `m` is nil
```suggestion
		if m, _ := kcmdline.KeyValues("ubuntu-core.force-experimental-tokens"); m != nil {
			val := m["ubuntu-core.force-experimental-tokens"]
			switch val {
			case "0":
				return false
			case "1"
				return true
			default:
				logger.Noticef("unexpected value for snapd.force-experimental-tokens: %q", val)
			}
		}

```